### ISSUE TITLE
pub breaks the layout of the package (macos)

### ISSUE THREAD

**Issue by zezo357** [alexmercerind/media_kit#104]
macos build fails with pub 1.0.1 and works with github 

<details>

<summary>Error log</summary>

Building macOS application...
In file included from /Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/media_kit_native_event_loop.cc:3:
In file included from /Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/../../src/media_kit_native_event_loop.cc:9:
/Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/../../src/media_kit_native_event_loop.h:17:10: fatal error: 'mpv/client.h' file not found
#include <mpv/client.h>
         ^~~~
1 error generated.
In file included from /Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/media_kit_native_event_loop.cc:3:
In file included from /Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/../../src/media_kit_native_event_loop.cc:9:
/Users/runner/hostedtoolcache/flutter/stable-3.7.9-x64/.pub-cache/hosted/pub.dev/media_kit_native_event_loop-1.0.2/macos/Classes/../../src/media_kit_native_event_loop.h:17:10: fatal error: 'mpv/client.h' file not found
#include <mpv/client.h>
         ^~~~
1 error generated.
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flutter Assemble' from project 'Runner')
 BUILD FAILED 

</details>

```
media_kit_libs_macos_video:
   git:
     url: https://github.com/alexmercerind/media_kit
     path: ./media_kit_libs_macos_video
```
<details>

<summary>Pod install Using github</summary>

![image](https://user-images.githubusercontent.com/25157308/229645015-52b72908-c0da-4338-8188-07ff0258b32f.png)

</details>

<details>

`media_kit_libs_macos_video: ^1.0.1`
<summary>Pod install Using pub</summary>

![image](https://user-images.githubusercontent.com/25157308/229645050-d47e1da3-09dd-48b6-ae9a-493c5dd818c4.png)

</details>

**Comment by birros** [comment]
I just checked, I confirm, **it is critical!**

I understood the problem, the dot files are not present in a pub package, I keep empty folders with `.gitignore` inside.

I create a pull request in emergency to fix the problem!

**Comment by birros** [comment]
Closed by #105
